man/systemd.unit.xml

```diff
@@ -1658,6 +1658,28 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>ConditionMemoryPressure=</varname></term>
+          <term><varname>ConditionCPUPressure=</varname></term>
+
+          <listitem><para>Verify that the overall system (memory or CPU) pressure is below the given threshold.
+          This setting takes a colon-separated tuple as argument. The first value is the slice unit under which
+          the pressure will be checked, and it is optional: if not specified, the overall system pressure will be
+          measured, instead of a particular cgroup's.</para>
```

Comment:
When there is a condition error (e.g. the provided slice doesn't exist) does the condition succeed or fail? IMO I think the condition should succeed if a slice doesn't exist (since no slice == no pressure)

Also I feel like this implies a dependency on certain systemd slices without providing any kind of actual dependency through the usual unit dependency mechanism.

Reply:
Due to the way conditions are set up, it would require quite a bit of lifting and refactoring to bring that information down to where it's checked. Also, I figured this allows more interesting patterns, like avoiding trampling over _other_ slices than your own - for example a task in the new desktop background.slice could be set up to be skipped if app.slice is under pressure.
But yes if it's missing it should not be an error, but it should pass.

Reply:
Changed as requested